dev/start

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -ex
+
+# Determine which extra services should be loaded when working with authenticators
```

Comment:
If we're going to use this as an example of how to write a `start` script,  we should use `getopt` for argument parsing: http://www.bahmanm.com/blogs/command-line-options-how-to-parse-in-bash-using-getopt .

Reply:
@apotterri (partial) counterpoint:
https://mywiki.wooledge.org/BashFAQ/035#Manual_loop

Reply:
That appears to be comparing it with `getopts` (with an `s`), which is older and less functional than `getopt`.  I believe all the criticisms they offer of `getopts` have now been addressed in the new command.

I think we use the standard thing, rather than rolling our own in each project.